webrender/src/device/gfx/device.rs

```diff
@@ -63,6 +63,14 @@ const COLOR_RANGE: hal::image::SubresourceRange = hal::image::SubresourceRange {
     layers: 0 .. 1,
 };
 
+#[derive(PartialEq)]
+pub enum BackendApiType {
+    Gl,
```

Comment:
Maybe we don't need this variant at all, since GL is behind a different feature.